testhelper/docker/resource/redis.go

```diff
@@ -0,0 +1,75 @@
+package resource
```

Comment:
since we are moving this to commons, it would be wise to add support for overriding the docker image tag.

Reply:
Good idea, will add that :+1:

Reply:
Done [here](https://github.com/rudderlabs/rudder-go-kit/pull/21/commits/7684c62ca562fa925cde6e4c10429325e358c73b).